handsontable/src/helpers/dom/element.js

```diff
@@ -769,7 +769,7 @@ export function getComputedStyle(element, rootWindow = window) {
  * @returns {number} Element's outer width.
  */
 export function outerWidth(element) {
-  return Math.ceil(element.getBoundingClientRect().width);
+  return element.offsetWidth;
```

Comment:
It's reverting the change introduced in #7281, but if I'm checking correctly, that issue seems no longer present in the current version of Safari. Probably worth double-checking during the freeze @aninde.